mail/templates/verification/body.html

```diff
@@ -1,11 +1,44 @@
 {% extends "email_base.html" %}
 
-{% block title %}Confirm your email address{% endblock %}
+{% block title %}Verify your email{% endblock %}
 
 {% block content %}
-<div class="section">
-  <p>Someone (hopefully you) attempted to register on Open Discussions with this email address.</p>
-  <p>To confirm, <a href="{{ confirmation_url }}">click here to verify your email address</a>.</p>
-  <p>Otherwise, you may safely ignore this email.</p>
-</div>
+<!-- 1 Column Text + Button : BEGIN -->
+  <tr>
+      <td style="background-color: #ffffff;">
+          <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+              <tr>
+                  <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
+                      <h1 style="margin: 0 0 10px; font-size: 25px; line-height: 30px; color: #03152d; font-weight: bold;">Verify your email</h1>
+                      <p style="margin: 0 0 10px;">Thank you for signing up for MIT Open. To finish signing up, you just need to confirm that we got your email right.</p>
```

Comment:
`Thank you for joining {{ site_name }}` instead of `Thank you for signing up for MIT Open`